src/app/components/elements/MarkNotificationRead.jsx

```diff
@@ -27,5 +27,5 @@ class MarkNotificationRead extends React.Component {
 }
 
 export default connect(null, dispatch => ({
-    update: (payload) => { dispatch({type: 'UPDATE_NOTIFICOUNTERS', payload})},
+    update: (payload) => { dispatch({type: 'app/UPDATE_NOTIFICOUNTERS', payload})},
```

Comment:
Does this deserve an action creator? à la `appActions.addNotification` ?

Reply:
oops it actually has one and i forgot to change this guy